internal/pkg/local/load.go

```diff
@@ -2,96 +2,98 @@ package local
 
 import (
 	"fmt"
-	"path/filepath"
-	"reflect"
-	"strings"
 
-	"github.com/keboola/keboola-as-code/internal/pkg/json"
+	"github.com/keboola/keboola-as-code/internal/pkg/mapper"
 	"github.com/keboola/keboola-as-code/internal/pkg/model"
-	"github.com/keboola/keboola-as-code/internal/pkg/transformation"
 	"github.com/keboola/keboola-as-code/internal/pkg/utils"
 	"github.com/keboola/keboola-as-code/internal/pkg/validator"
 )
 
-// LoadModel from manifest and disk.
-func (m *Manager) LoadModel(record model.Record, target interface{}) (found bool, err error) {
-	errors := utils.NewMultiError()
+type files = model.ObjectFiles
 
-	// Check if directory exists
-	if !utils.IsDir(filepath.Join(m.ProjectDir(), record.RelativePath())) {
-		errors.Append(fmt.Errorf(`%s "%s" not found`, record.Kind().Name, record.RelativePath()))
-		return false, errors
+type modelLoader struct {
+	*Manager
+	*files
+	mapper *mapper.Mapper
```

Comment:
Okrem upravy na abstraktny FS som pridal `mapper` vrstvu, ktora nam pomoze s implementaciou dalsich features.

Reply:
`Mapper` vrstvu som ma implementovanu skor, ako som zacal riesit filesystem, preto je to v jednom PR.
Cely load ma iba nejakych `90` riadkov.